src/interpreter/index.ts

```diff
@@ -77,6 +81,16 @@ export class Interpreter implements Expr.Visitor<BrsType>, Stmt.Visitor<BrsType>
         return this._environment;
     }
 
+    public setCoverageCollector(collector: CoverageCollector) {
+        this.coverageCollector = collector;
+    }
+
+    public reportCoverageHit(statement: Expr.Expression | Stmt.Statement) {
```

Comment:
I may be missing something, but I suspect this doesn't strictly need to be `public` right?